redis/cache/redispipebp/monitored.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/reddit/baseplate.go/prometheusbp"
 	"github.com/reddit/baseplate.go/redis/cache/redisx"
+	"github.com/reddit/baseplate.go/redis/redisprom"
```

Comment:
🔕 as I don't feel strongly, but I would prefer to just define the metrics following the spec twice, in each client packages, instead of providing a package just for the metric definitions.

but if we are going to provide a package, it probably should be `internal` (either `baseplate.go/internal/redisprom`, or `baseplate.go/redis/internal/redisprom`)

Reply:
moved to internal